pkg/cloudprovider/cloudprovider.go

```diff
@@ -163,7 +163,7 @@ func getCABundle(ctx context.Context, restConfig *rest.Config) (*string, error)
 	if err != nil {
 		return nil, fmt.Errorf("discovering caBundle, loading TLS config, %w", err)
 	}
-	logging.FromContext(ctx).Debugf("Discovered caBundle, length %d", len(transportConfig.TLS.CAData))
+	logging.FromContext(ctx).With("cBundle-count", len(transportConfig.TLS.CAData)).Debugf("discovered caBundle")
```

Comment:
cBundle is a typo. Also -- I'm not sure we actually care about the length. You can just remove, I think :)